extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -161,38 +154,39 @@ public static class Secret {
 
             public static enum Method {
                 /**
-                 * client_secret_basic (default): client id and secret are submitted with the HTTP Authorization Basic scheme
+                 * `client_secret_basic` (default): The client id and secret are submitted with the HTTP Authorization Basic
+                 * scheme.
                  */
                 BASIC,
 
                 /**
-                 * client_secret_post: client id and secret are submitted as the `client_id` and `client_secret` form
-                 * parameters.
+                 * `client_secret_post`: The client id and secret are submitted as the `client_id` and `client_secret`
+                 * form parameters.
                  */
                 POST,
 
                 /**
-                 * client_secret_jwt: client id and generated JWT secret are submitted as the `client_id` and `client_secret`
-                 * form
-                 * parameters.
+                 * `client_secret_jwt`: The client id and generated JWT secret are submitted as the `client_id` and
+                 * `client_secret`
+                 * form parameters.
                  */
                 POST_JWT
             }
 
             /**
-             * The client secret value - it will be ignored if 'credentials.secret' is set
+             * Optional. The client secret value. This value is ignored if `credentials.secret` is set.
```

Comment:
Suggest to update similarly to have a secret property above is proposed to be updated

Reply:
Thanks. Updated. Is this what you mean?
```suggestion
             * The client secret value. This value is ignored if `credentials.secret` is set.
             * Must be set unless a secret is set in {@link #clientSecret} or {@link #jwt} client authentication is required.```